### DISCUSSION TITLE
Already upgraded Copilot (student) and use it on my second computer, but getting "Please upgrade your Copilot extension to continue using this service." on my main computer.

### DISCUSSION THREAD

**Discussion by sarah7sami** [community/community#33431] [Copilot Conversations] [answered]
To expand on the issue, I have been using copilot on my other computer with the same GitHub account with no issue.
I activated it on this laptop, and as many times as I reload, redownload, and reauthorize, I still get: 

`[INFO] [default] [2022-09-18T16:16:04.012Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-09-18T16:16:04.041Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 29 ms
[INFO] [fetch] [2022-09-18T16:16:04.042Z] Please upgrade your Copilot extension to continue using this service.`

In the output log. It even worked one time after many many attempts at reloading, so I believe use on 2 devices is not the problem. 
Any suggestions on how to fix this?

**Comment by davecheney** [comment, marked as answer]
@sarah7sami please see https://github.com/community/community/discussions/30716
